lib/src/__tests__/e2e/DateTimePickerRoot.test.tsx

```diff
@@ -70,3 +70,33 @@ describe('e2e - DateTimePicker', () => {
     toHaveBeenCalledExceptMoment(onChangeMock, [utilsToUse.date('2018-01-01T12:00:00.000Z')]);
   });
 });
+
+describe('e2e - DateTimePicker with seconds', () => {
+  let component: ReactWrapper<DateTimePickerProps>;
+  const onChangeMock = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    component = mount(
```

Comment:
Please prefer testing-library for this

Reply:
I've based my test on existing tests which are using enzyme, is it prefered that the tests use testing-library over being consistent with the existing tests?

Reply:
@JamesMHenderson the objective is to, step by step, be all in on testing-library. It requires all new tests to use it. This holds for all our codebase (core, pickers, etc). For instance, on the pickers side https://github.com/mui-org/material-ui-pickers/issues/1933.